Codelab/MusicMaker-step2-Scripting/src/main/assets/sound_blocks.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "type": "play_sound",
+    "message0": "Play %1",
+    "args0": [{
+      "type": "field_dropdown",
+      "name": "VALUE",
+      "options": [
+        ["Cheer", "sounds/yeah.mp3"],
```

Comment:
nit: The other two codelabs leave these extra sounds out of the step-by-step and suggest adding another block for them as a way to explore beyond the codelab. Though, they are included in the completed demo on iOS.